### ISSUE TITLE
Segmentation fault on musl libc

### ISSUE THREAD

**Issue by begss** [ppy/osu#12319]
**Describe the bug:** 
Dotnet officially supports musl and Microsoft makes builds for Alpine linux, which are actually builds for any musl-based distro. Osu compiles successfully, but segfaults when attempting to execute `osu.Desktop/bin/Release/net5.0/osu!`. Generally reproducible on any hardware and any musl-based linux distribution, but I'm using Void Linux x86_64.

**Strace:** 
[strace.txt](https://github.com/ppy/osu/files/6271451/strace.txt)

**osu!lazer version:** 
git head

**Comment by bdach** [comment]
dotnet supporting musl does not mean the game does. in this case it seems that the segfault happens somewhere in input device handling, so it'll 99% be in scope of either sdl2 or opentabletdriver.

closing as low priority/out of scope. contributions to fix are appreciated.

**Comment by begss** [comment]
As I'm not a C# programmer, I very roughly understand the codebase of osu, but I made some investigation regarding to libs.

I located `libSDL2.so` at `osu.Desktop/bin/Release/net5.0/runtimes/linux-x64/native/libSDL2.so`. Then I found almost similar directory two levels up, `../../linux-musl-x64/native`, where was only `libe_sqlite3.so` and no other libs.

Then I replaced `libSDL2.so` with the system one, taken from `/usr/lib/libSDL2-2.0.so.0`. Segfaults again, but I could hear that SDL tried to connect sound server. Then I replaced `libe_sqlite3.so` with one from `../../linux-musl-x64` and osu launched properly (yay).

So as I think the problem is that `linux-musl-x64` runtime directory is not used at all, and also missing native builds of most libs for musl.
